.github/workflows/publish-to-other-than-github.yaml

```diff
@@ -130,24 +130,11 @@ jobs:
       run: |
         $version = "${{ github.event.client_payload.version }}" -replace 'v'
         echo "RELEASE_VERSION=$version" | Out-File $env:GITHUB_ENV
-    - name: Generate token
-      id: generate_token
-      uses: tibdex/github-app-token@v2
-      with:
-        app_id: ${{ secrets.OCMBOT_APP_ID }}
-        private_key: ${{ secrets.OCMBOT_PRIV_KEY }}
-        permissions: |
-          {"contents":"write","pull_requests":"write"}
-    - name: Checkout
-      uses: actions/checkout@v4
-      with:
-        token: ${{ steps.generate_token.outputs.token }}
     - name: Install wingetcreate
-      run: choco install wingetcreate
+      run: iwr https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
     - name: Update Winget package
       run: |
-        cd hack\winget
```

Comment:
do we need that repository?

Reply:
actually I'm not 100% sure, it was necessary to create the very first version, but I hope the update is working without it... then we can also remove the complete folder.